scram2cmake.pl

```diff
@@ -16,11 +16,14 @@
 my $prods="${base}/.SCRAM/${arch}/ProjectCache.db.gz";
 chdir($base);
 system("rm -rf $proj_modules $tools; mkdir -p $proj_modules");
+system("${SCRIPT_DIR}/runtime2cmake.pl $proj_modules");
 if ($proj eq "")
 {
   print "Generating tools...\n";
   system("${SCRIPT_DIR}/tools2cmake.pl $tools");
-  system("${SCRIPT_DIR}/runtime2cmake.pl $proj_modules");
```

Comment:
@gartung , keep this here to avoid running it twice (once for cmssw and once for coral).